crates/transparency/benches/map.rs

```diff
@@ -2,7 +2,7 @@ use std::{iter::repeat_with, time::Duration};
 
 use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
 use rand::Rng;
-use warg_crypto::hash::Sha256;
+use warg_crypto::hash::{AnyHash, Hash, Sha256};
```

Comment:
I think this change can be backed out.

Reply:
Looks like we are left with a change to add brackets around `Sha256`, can probably just revert the whole file.